sql/parser/create.go

```diff
@@ -276,7 +279,10 @@ func (node *ColumnTableDef) Format(buf *bytes.Buffer, f FmtFlags) {
 			buf.WriteByte(')')
 		}
 	}
-
+	if node.Family != "" {
```

Comment:
One option here might be to kick this out of `col_qualification_elem` and throw it directly into `col_qualification`, to avoid making it nameable in the syntax.